test/azure/Expected/AcceptanceTests/AzureSpecials/setup.py

```diff
@@ -19,7 +19,7 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["msrest>=0.6.0", "azure-core<2.0.0,>=1.2.0"]
+REQUIRES = ["msrest>=0.6.0", "azure-core<2.0.0,>=1.2.0", "azure-mgmt-core>=1.2.0"]
```

Comment:
lowest version is 1.2.0 since we're using the new `http_logging_policy` property (changelog [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-mgmt-core/CHANGELOG.md#features))